packages/snjs/lib/application.ts

```diff
@@ -340,6 +346,11 @@ export class SNApplication {
     // optional override
   }
 
+  public getAppVersion(): string {
+    // mandatory override
+    return '';
```

Comment:
Should throw exception if mandatory override.
```suggestion
    throw Error('getAppVersion must be overriden.');
```

Reply:
I actually removed this method at all since I'm already passing version in constructor, just didn't push yet. But really good approach for such cases :)